client/lib/post-normalizer/index.js

```diff
@@ -469,6 +471,32 @@ normalizePost.withContentDOM = function( transforms ) {
 
 normalizePost.content = {
 
+	createContentWithLinebreakElementsOnly: function createContentWithLinebreakElementsOnly( post, callback ) {
+		if ( ! post.__contentDOM ) {
+			throw new Error( 'this transform must be used as part of withContentDOM' );
+		}
+
+		post.content_with_linebreak_elements_only = striptags( post.__contentDOM.innerHTML, [ 'p', 'br', 'noscript' ] );
```

Comment:
Yep. I think we can drop this now we're running `createContentWithLinebreakElementsOnly` after `detectPolls` in the post normalizer.